src/Akka.Logger.Serilog/SerilogLoggingAdapterExtensions.cs

```diff
@@ -30,7 +30,7 @@ public static ILoggingAdapter GetLogger<T>(this IActorContext context)
             var logSource = context.Self.ToString();
             var logClass = context.Props.Type;
 
-            return Activator.CreateInstance(typeof(T), new BusLogging(context.System.EventStream, logSource, logClass, new DefaultLogMessageFormatter())) as T;
```

Comment:
So this method, ` public static ILoggingAdapter GetLogger<T>(this IActorContext context)` - maybe it should be renamed. I don't know why it was generic in the first place given that nothing other than `Akka.Logger.Serilog` could access it.

IMHO, we could rename it and break this method on purpose to get everyone who is using the semantic logging syntax to explicitly say "yes, I am using semantic logging and I am going to use the correct `ILoggingAdapter` implementation accordingly." Or leave this method signature as-is and not break anything. I defaulted to the latter but considered the former. What do you think @Danthar ?

Reply:
The generic type doesn't do anything, other than provide a unique method signrature and give the reading a hint as to whats being returned. 
The unique signature is needed because `IActorContext.GetLogger()` already exists afaik

Reply:
Ok cool, I'll leave it as-is for now.